wrappers/python/simtk/openmm/app/pdbfile.py

```diff
@@ -237,7 +241,7 @@ def _parseResidueAtoms(residue, map):
                 map[atom.attrib[id]] = name
 
     @staticmethod
-    def writeFile(topology, positions, file=sys.stdout, keepIds=False):
+    def writeFile(topology, positions, file=sys.stdout, keepIds=False, extraParticleIdentifier=' '):
```

Comment:
Check out the discussion above where @peastman points out we need to write out PDB files according to the PDB specification - i.e. `None` element needs to be written out as blank. If you want to write your own-style PDB file that does not conform to specification, you'll have to set this to 'EP' yourself.